src/SketchExample.js

```diff
@@ -1,49 +1,48 @@
-import React, { Component } from "react";
+import React, { useState } from "react";
 import { SketchPicker } from "react-color";
 import "./SketchExample.css";
 import PropTypes from "prop-types";
-class SketchExample extends Component {
-  constructor(props) {
-    super(props);
-    this.state = {
-      displayColorPicker: false,
-      background: this.props.background
-    };
-  }
 
-  handleChange = () => {
-    this.setState({
-      displayColorPicker: !this.state.displayColorPicker
+const SketchExample = props => {
+  let colorItem = {
+    displayColorPicker: false
+  };
+
+  const [colorData, setData] = useState({
+    colorItem
```

Comment:
do not use colorItem variable. we can use this.
 useState({
    displayColorPicker: false
  }) ...